package.json

```diff
@@ -14,20 +14,21 @@
         "tls": false
     },
     "dependencies": {
+        "browserify": "^17.0.0",
```

Comment:
This is a devDependency not a runtime dependency

Reply:
Yes. Missed that it had got there. I found in command history I had run `npm upgrade browserify`. It did not have version before, so not sure why I had done that. If npm warned it required `--dev` option, I missed it too.

I have an old draft PR that contained a debug upgrade commit. I'll toss that in as well. No reason to keep old debug for node 4 support.